gateway/ln-gateway/tests/fixtures.rs

```diff
@@ -1,39 +1,35 @@
+use std::sync::Arc;
+
 use fedimint_dummy_client::DummyClientGen;
 use fedimint_dummy_common::config::DummyGenParams;
 use fedimint_dummy_server::DummyGen;
 use fedimint_ln_client::LightningClientGen;
 use fedimint_ln_common::config::LightningGenParams;
 use fedimint_ln_server::LightningGen;
-use fedimint_testing::btc::mock::FakeBitcoinTest;
 use fedimint_testing::btc::BitcoinTest;
 use fedimint_testing::federation::FederationTest;
 use fedimint_testing::fixtures::Fixtures;
 use fedimint_testing::gateway::GatewayTest;
 use ln_gateway::rpc::rpc_client::GatewayRpcClient;
 
-pub async fn fixtures(
-    password: Option<String>,
-) -> (
+/// Constructs a gateway connected to 2 federations for RPC tests
+pub async fn fixtures() -> (
     GatewayTest,
     GatewayRpcClient,
     FederationTest,
     FederationTest,
-    Box<dyn BitcoinTest>,
+    Arc<dyn BitcoinTest>,
 ) {
-    // TODO: use new client modules without legacy instances
-    let mut fixtures =
-        Fixtures::new_primary(1, DummyClientGen, DummyGen, DummyGenParams::default());
+    let mut fixtures = Fixtures::new_primary(DummyClientGen, DummyGen, DummyGenParams::default());
     let ln_params = LightningGenParams::regtest(fixtures.bitcoin_server());
-    fixtures = fixtures.with_module(0, LightningClientGen, LightningGen, ln_params);
+    fixtures = fixtures.with_module(LightningClientGen, LightningGen, ln_params);
 
-    let gateway = fixtures.new_gateway(password).await;
+    let lnd = fixtures.lnd().await;
+    let gateway = fixtures.new_gateway(lnd).await;
```

Comment:
I forget when this is used. Is `lnd` always the gateway when this is called?

Reply:
So `lnd()` and `cln()` will give you the two kinds of test LN nodes

You can then pass them into as many gateways as you wish.  Then connect gateways to as many feds as you wish.